### PR TITLE
Updating JWPL Dependencies - Colon URIs

### PR DESCRIPTION
**Pull request by dav009**
- Added the new JWPL parser version dep
- Added a test to make sure we are getting uris with colons on jsonpedia
- Sacked off what seems to be unused dependencies

Can someone get this branch and do `mvn package` to confirm I did not jinx the dependencies. 
-- edit --
confirmed by @llu that it is not jinxed

fixes https://github.com/idio/json-wikipedia/issues/7

Thanks

### CODE REVIEW

**Comment by keynmol** [comment]
The way we version manage this literally makes me puke in my mouth :(

**Comment by dav009** [comment]
yeah, we should merge this two projects as soon as possible.
I just battled `mvn install` for 30mins @_@

**Comment by keynmol** [comment]
*sigh*

File: src/test/java/it/cnr/isti/hpc/wikipedia/article/en/ArticleTest.java [hunk 1]
```diff
@@ -185,7 +185,7 @@ public void testAnnotationsInLists() throws IOException {
 			}
 		}
 
-		assertThat(uris, hasItems("Yū_Hayami", "Takumi_Hayami", "Hayami_District,_Ōita", "Mokomichi_Hayami"));
+		assertThat(uris, hasItems("Yū_Hayami", "Takumi_Hayami", "Hayami_District,_Ōita", "Mokomichi_Hayami", "H2O:_Footprints_in_the_Sand"));
```
Comment (Lugrin) [inline src/test/java/it/cnr/isti/hpc/wikipedia/article/en/ArticleTest.java]: How come there is a new item? Did you change the file `./src/test/resources/en/Hayami`?

Comment (dav009) [inline src/test/java/it/cnr/isti/hpc/wikipedia/article/en/ArticleTest.java]: No.

That item has been there all the time, I just did not check its extraction in the test, cause I was 100% sure that it was failing.
So with the fixest that this PR introduces, this URI should be def extracted, thats why I added it to the test.

Comment (dav009) [inline src/test/java/it/cnr/isti/hpc/wikipedia/article/en/ArticleTest.java]: check the issue : https://github.com/idio/json-wikipedia/issues/7

**Pull request merged**